Control coverage/Feature/KeyVault.md

```diff
@@ -0,0 +1,5 @@
+| ControlId | Dependent Azure API(s) and Properties | Control Spec |
```

Comment:
A general comment on all updates - It is usually a "Spec-let" that is included here and not the actual "Spec". While the additional information helps, you may want to include just the "Spec-let" to be uniform with what is currently available.